app/controllers/application_controller.rb

```diff
@@ -139,8 +138,4 @@ def meta_tag_description
   def requested_organisation_path
     organisation_path(Organisation.find(current_user.pending_organisation_id))
   end
-
-  def user_requested_organisation_admin_status?
-    current_user.pending_organisation_id
-  end
 end
```

Comment:
I removed this method because then the Code Climate was showing "Class has too many lines" offence. See https://codeclimate.com/github/AgileVentures/LocalSupport/pull/460

If you recommend split this class to multiple lines let me know.